save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/ReactorUtils.kt

```diff
@@ -277,6 +277,18 @@ fun <T> Mono<T>.switchIfErrorToConflict(
     messageCreator: () -> String? = { null },
 ) = switchIfErrorToResponseException(HttpStatus.CONFLICT, messageCreator)
 
+/**
+ * @param function blocking operation like JDBC
+ * @return [Mono] from result of blocking operation [R]
+ * @see blockingToMono
+ * @see ResponseSpec.blockingBodyToMono
+ * @see ResponseSpec.blockingToBodilessEntity
+ * @see BlockingBridge
+ */
+fun <T : Any, R : Any> Mono<T>.blockingMap(function: Function1<T, R>): Mono<R> = flatMap { value ->
```

Comment:
Is `Function1<T, R>` the same as `(T) -> R`?

Reply:
yes, it is

Reply:
refactored to more readable format